features/namespace.feature

```diff
@@ -3,7 +3,7 @@ Feature: Interact with the Namespace API
   Scenario: Try retrieve a namespace that does not exist
     Given a namespace called rain does not exist
       When the user attempts to retrieve the namespace rain
-        Then None is returned
+        Then None is returned for the namespace
```

Comment:
`Then None is returned instead of the namespace`

Reply:
@pmyjavec Thanks for the comments. I have addressed the all the comments. Please  review